include/z64audio.h

```diff
@@ -0,0 +1,1208 @@
+#ifndef Z64_AUDIO_H
+#define Z64_AUDIO_H
+
+#define MK_CMD(b0,b1,b2,b3) ((((b0) & 0xFF) << 0x18) | (((b1) & 0xFF) << 0x10) | (((b2) & 0xFF) << 0x8) | (((b3) & 0xFF) << 0))
```

Comment:
Is this a shorthand for "Make command"?
I think it kinda needs something in the name to make it clear it is audio related.
I have no idea how it is used, but how about `MK_AUDIO_CMD`?

Reply:
Seems like this macro is only used a single time. There might be a better way to deal with commands, but for now yeah I'll switch it to `MK_AUDIO_CMD`.